alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -17,9 +15,13 @@ use windows_sys::Win32::System::Threading::{
 
 use crate::config::PtyConfig;
 use crate::event::{OnResize, WindowSize};
+use crate::tty::windows::blocking::{UnblockedReader, UnblockedWriter};
 use crate::tty::windows::child::ChildExitWatcher;
 use crate::tty::windows::{cmdline, win32_string, Pty};
 
+// Adjust to taste. Too high means higher memory consumption, too low means over-synchronization.
+const PIPE_CAPACITY: usize = 4 * 1024 * 1024;
```

Comment:
Why that particular value was picked? We have a `READ_BUFFER_SIZE` inside the event_loop, so maybe it should be tuned based on it?

Reply:
It's the default pipe capacity used in the `blocking` crate. I've fixed it to just equal `READ_BUFFER_SIZE`.

Reply:
Maybe it should be benchmarked, with e.g. vtebench, but from what I remember conpty is so slow that it doesn't matter what we pick.